.puppet-lint.rc

```diff
@@ -1 +1,2 @@
 --relative
+--no-140chars-check
```

Comment:
This is included in the PDK template (https://github.com/puppetlabs/pdk-templates/blob/main/moduleroot/.puppet-lint.rc.erb) without a matching `.sync.yml` entry. That means it'll be overridden on the next update.

Reply:
Just a quick Question:  Should I `git commit --amend && git push -f` the Changes, or should I append another Commit?

Reply:
I added the Check to `.sync.yml`, too.